packages/solecs/package.json

```
@@ -10,8 +10,9 @@
     "directory": "packages/solecs"
   },
   "scripts": {
+    "clean": "rimraf out && rimraf dist && rimraf cache",
```

I don't think `solecs` has a `dist` folder, but there are a couple other folders we could clean here:

```suggestion
    "clean": "rimraf out && rimraf cache && rimraf types && rimraf abi",
```

(`types` and `abi` is already cleaned by calling `dist` and `types` though, so not sure if relevant here)

I guess the main purpose of this is to remove the forge output/cache, so maybe we could just call it `clean:forge` and use the built-in `forge clean`?

```suggestion
    "clean:forge": "forge clean",
```